src/modules/spaceships.js

```diff
@@ -0,0 +1,31 @@
+import spaceship1 from '../images/spaceship1.png';
+import spaceship2 from '../images/spaceship2.png';
+import spaceship3 from '../images/spaceship4.png';
+
+function createSpaceships() {
```

Comment:
Kindly use ES6 syntax for your application.  This means using `arrow functions` instead of the `function` keyword. 

Your function should be `const createSpaceships = () => {...};`

- Kindly apply that to all your files including the `showNotification`, `mobileMenu`, `insertLogo` functions etc.